circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 0.10.0
```

Comment:
The problem is that we are moving from Travis to CircleCI, but it seems that it does not support multiple versions in the build. We are investigating if CircleCI has some workaround to test with multiple node versions.